flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -496,7 +503,12 @@ public final class ConfigConstants {
      */
     @Deprecated public static final String YARN_TASK_MANAGER_ENV_PREFIX = "yarn.taskmanager.env.";
 
-    /** Template for the YARN container start invocation. */
+    /**
+     * Template for the YARN container start invocation.
+     *
+     * @deprecated in favor of {@code YarnConfigOptions#YARN_CONTAINER_START_COMMAND_TEMPLATE}.
```

Comment:
```suggestion
     * @deprecated in favor of {@link YarnConfigOptions#YARN_CONTAINER_START_COMMAND_TEMPLATE}.
```

nit: `@link` is easy to forward in IDEA.

Reply:
I tried before, but it doesn't work. The reason is that flink-core module doesn't depend on flink-yarn module, so we cannot link to `YarnConfigOptions#YARN_CONTAINER_START_COMMAND_TEMPLATE`, which is defined in flink-yarn module. 

YARN_APPLICATION_MASTER_PORT has the same problem, so it uses `@code` as weel.